Makefile

```diff
@@ -108,8 +108,9 @@ endif
 GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
 
 CROSS_BUILD_TARGETS := \
-	bin/podman.cross.darwin.amd64 \
```

Comment:
Are we dropping the Darwin target?

Reply:
Ahhh, is it part of the `podman-rmote-darwin` build above?

Reply:
that can NEVER work ... in order for it to work, it must be the darwin REMOTE client.